code/__DEFINES/inventory.dm

```diff
@@ -134,7 +134,9 @@
 #define HEADCOVERSMOUTH (1<<4)
 #define PEPPERPROOF (1<<5) //protects against pepperspray
 
-#define TINT_DARKENED 2 //Threshold of tint level to apply weld mask overlay
+#define TINT_NONE 0.25
```

Comment:
Perhaps you should call this `TINT_MINIMAL` hecause it's not zero.

Reply:
yer right